ekip/everykid/static/css/voucher.css

```diff
@@ -31,7 +31,7 @@ html, body {
 
 .voucher-container, .voucher-instructions {
   width: 610px;
-  height: 842px;
+  height: 880px;
```

Comment:
The Spanish text takes more space. With the 842px height, the text was running out of the border like this:
<img width="656" alt="screen shot 2016-11-30 at 10 44 37 am" src="https://cloud.githubusercontent.com/assets/840844/20759127/092395f0-b6ea-11e6-856f-10cc977ca6c1.png">